pkg/clients/unstructured_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2016-2019, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clients
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/pulumi/pulumi-kubernetes/pkg/await/fixtures"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func TestFromUnstructured(t *testing.T) {
```

Comment:
Are there plans to test the rest of the supported objects other than the Pod and Deployment?

Reply:
Yeah, probably should have been more clear in the description, but this PR is basically a prototype to make sure the approach made sense before I fully committed to it. My intent is to expand these fixtures to replace all the giant YAML strings in the unit tests. I'll keep fleshing things out over multiple PRs.